docs/introduction.html

```diff
@@ -18,7 +18,7 @@
 <script><!--#include virtual="js/templateData.js" --></script>
 
 <script id="introduction-template" type="text/x-handlebars-template">
-  <h3> Kafka is <i>a distributed streaming platform</i>. What exactly does that mean?</h3>
+  <h3> Kafka&trade; is <i>a distributed streaming platform</i>. What exactly does that mean?</h3>
```

Comment:
I'm going to start off with saying I'm not a trademark expert, but from what I understand, the rule is that the first instance of "Apache Kafka" or "Kafka" needs to have the trademark symbol on it. The "Apache Kafka" logo at the top of every page of the site has the TM symbol on it so my interpretation of the TM requirement is that we don't need to ALSO add the TM to the first instance of "Kafka" or "Apache Kafka" in body content. You might want to double check with @junrao or Vivian from marketing to get a definite answer on this.